### ISSUE TITLE
"it should run out of the box on any machine under Windows (without any requirement)" inaccuracy

### ISSUE THREAD

**Issue by chrisjj** [winpython/winpython#1335]
https://sourceforge.net/projects/winpython/ says:

"it should run out of the box on any machine under Windows (without any requirement)"

whereas https://winpython.github.io/ shows requirements include Window 7/8+.

**Comment by nmarc** [comment]
I don't fully understand. By '+' it means 'or higher'. What is the specific
problem?

Am Mi., 8. Mai 2024 um 14:13 Uhr schrieb chrisjj ***@***.***>:

**Comment by chrisjj** [comment]
The problem is "it should run out of the box on any machine under Windows
(without any
> requirement)" is untrue e.g. Windows below 7 - and on the latest version,
below 8.

+ makes no difference to that.

**Comment by nmarc** [comment]
I believe the initial statement refers to all current systems.
After all, no ordinary person nowadays uses an XP or Win3.x system and
still expects modern projects to run smoothly right out off the box.
I consider that to be a quibble. I've never had any issues with Winpython
on any Windows system (7,8,10,11) , and I am extremely grateful that true
professionals have taken on this project.


Am Mi., 8. Mai 2024 um 15:47 Uhr schrieb chrisjj ***@***.***>:

> The problem is "it should run out of the box on any machine under Windows
> (without any
> > requirement)" is untrue e.g. Windows below 7 - and on the latest
> version,
> below 8.
>
> + makes no difference to that.
>

**Comment by stonebig** [comment]
The old welcome page on sourceforge is a shrine of the initial glorious times of windows xp to 7... new, maintained, welcome page is winpython.github.io

**Comment by stonebig** [comment]
For awindows of the time, download a winpython version of the time, and it shall work as in the vintage advertisement

**Comment by chrisjj** [comment]
> I believe the initial statement refers to all current systems.

No, it does not refer to all current systems. It refers to Windows, unqualified.

Perhaps it should refer to all current systems. And state what it considered "current".

> After all, no ordinary person nowadays uses ...

WinPython is only for ordinary people, then? I'd better stop using it immediately! :)

>  I've never had any issues with Winpython on any Windows system (7,8,10,11) ,

I guess that's because you've not tried the latest WinPython on Windows 7. Here it won't run.

**Comment by chrisjj** [comment]
> The old welcome page on sourceforge is a shrine of the initial glorious times of windows xp sp2 to 7...

It shows activity one week ago and is distributing latest versions.

**Comment by stonebig** [comment]
hum, yes, still use it as plan b for downloads
